apps/chatbot/docker/app.local.Dockerfile

```diff
@@ -14,4 +18,10 @@ RUN poetry install
 
 COPY . .
 
-CMD ["fastapi", "dev", "src/app/main.py", "--port", "8080"]
+RUN python -m spacy download it_core_news_md
+RUN python -m spacy download en_core_web_md
```

Comment:
here are missing to download: `es_core_news_md`, `de_core_news_md`, `fr_core_news_md`,  `el_core_news_md`, `hr_core_news_md`, and `sl_core_news_md`.